Lombiq.Privacy.Samples/Recipes/Lombiq.Privacy.Samples.Content.recipe.json

```
@@ -0,0 +1,504 @@
+                      "Author": "admin",
+                      "FormInputElementPart": {
+                        "Name": "e-mail"
+                      },
+                      "FormElementPart": {
+                        "Id": "e-mail"
+                      },
+                      "InputPart": {
+                        "Type": "email",
+                        "DefaultValue": null,
+                        "Placeholder": "Your e-mail address"
+                      },
+                      "FlowMetadata": {
+                        "Alignment": 3,
+                        "Size": 100
+                      }
+                    },
+                    {
+                      "ContentItemId": "4pmfedtx6376bw44qbk063rqx8",
+                      "ContentItemVersionId": null,
+                      "ContentType": "PrivacyConsentCheckbox",
+                      "DisplayText": "",
+                      "Latest": false,
+                      "Published": false,
+                      "ModifiedUtc": "2022-05-12T19:00:59.8117192Z",
+                      "PublishedUtc": null,
+                      "CreatedUtc": null,
+                      "Owner": "",
+                      "Author": "admin",
+                      "PrivacyConsentCheckboxPart": {
+                        "ConsentCheckbox": false,
+                        "ShowAlways": true
+                      },
+                      "FlowMetadata": {
+                        "Alignment": 3,
+                        "Size": 100
+                      }
+                    },
+                    {
+                      "ContentItemId": "4qn4t1wngqc5j4vvggk1ngae5e",
+                      "ContentItemVersionId": null,
+                      "ContentType": "Button",
+                      "DisplayText": null,
+                      "Latest": false,
+                      "Published": false,
+                      "ModifiedUtc": "2022-05-12T19:00:59.8155113Z",
+                      "PublishedUtc": null,
+                      "CreatedUtc": null,
+                      "Owner": "",
+                      "Author": "admin",
+                      "FormInputElementPart": {
+                        "Name": null
+                      },
+                      "FormElementPart": {
+                        "Id": null
+                      },
+                      "ButtonPart": {
+                        "Text": "Register",
+                        "Type": "submit"
+                      },
+                      "FlowMetadata": {
+                        "Alignment": 3,
+                        "Size": 100
+                      }
+                    }
+                  ]
+                },
+                "FlowMetadata": {
+                  "Alignment": 3,
+                  "Size": 100
+                }
+              }
+            ]
+          }
+        }
+      ]
+
+    },
+    {
+      "name": "Settings",
+      "RegistrationSettings": {
+        "UsersCanRegister": 1,
```

Why is this `1` instead of `true`?

---

`UserCanRegister` is an enum and not a boolean property.
See in [RegistrationSettings.cs](https://github.com/OrchardCMS/OrchardCore/blob/62eed89890b6b9b6d71a02e1465269b9bb512207/src/OrchardCore.Modules/OrchardCore.Users/Models/RegistrationSettings.cs#L5)

---

I see, that makes sense. However you can set an enum with string too which is much less cryptic. I have just verified that this works:
```suggestion
        "UsersCanRegister": "AllowRegistration",
```

---

You are right.